docs/templates/finops-hub-with-exports.md

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
In the readme for templates, it says all templates should be placed within their own folder. Not sure if we care necessarily but should update one or the other.

Reply:
Never mind saw someone already commented on that